voorhoede/head-start
# Fix/142 xlinkhref is deprecated

# Changes

- Adds `href` to the use element. Kept `xlink:href` as fallback.
- Also adds some suggestions for improving accessibility.

# Associated issue

Resolves #142 

# How to test

1. Run dev server
2. Check share link
3. Icon works 🎉 

# Checklist
